### ISSUE TITLE
Document that modifications of structure of attributes table are not allowed

### ISSUE THREAD

**Issue by gioman** [boundlessgeo/qgis-geogiglight-plugin#48]
This case is correctly caught

```
The structure of attributes table has been modified.
This type of change is not supported by GeoGig.
```

nevertheless is better do document this to avoid that users will have to find them the hard way, and than they will have to revert a bunch of local changes.

I can take care of this.

**Comment by volaya** [comment]
I have tried to assign this to yourself, but I can't. Permissions issue?

**Comment by gioman** [comment]
@volaya I don't know, who manages the BL GH repo?

**Comment by gioman** [comment]
see https://github.com/boundlessgeo/qgis-geogiglight-plugin/pull/62

**Comment by alexbruy** [comment]
Closed in af91453bfe
